aws/data_source_aws_iot_endpoint_test.go

```diff
@@ -0,0 +1,27 @@
+package aws
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+func TestAccAWSIotEndpointDataSource(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: testAccAWSIotEndpointConfig,
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttrSet("data.aws_iot_endpoint.example", "id"),
```

Comment:
Sorry I missed this the first review, we don't need to explicitly check `id`, its implicitly expected to be there as part of being a Terraform resource/data source. I'll remove this on merge 👍